noaa/operations/default.json

```diff
@@ -1128,6 +1128,28 @@
         }
       }
     },
+    {
+      "name": "range-aggregation",
+      "operation-type": "search",
+      "body": {
+        "size": 0,
+        "aggs": {
+          "tmax": {
+            "range": {
+              "field": "TMAX",
+              "ranges": [
+                {"to":   -10},
+                {"from": -10, "to":  0},
```

Comment:
Nit: Can we standardize on the JSON format we used in big5? 
From
```
 {"from":   0, "to": 10},
```
To
```
 {
     "from":   0, 
     "to": 10
},
```

Reply:
In this json, the format is like this. I think if we want to format, better to do in a different PR.

Reply:
Makes sense. I see it in the subsequent operations.